src/components/common/Input/Input.tsx

```diff
@@ -0,0 +1,9 @@
+import type { FC } from 'react';
+import css from './styles.module.css'
+
+interface InputProps {
+    value: string;
+    type?: 'addTask';
+}
+
+export const Input: FC<InputProps> = ({ value, type='addTask' }) => <input value={ value } className={ css[type] } />
```

Comment:
тип был для примера, тут не нужен тип, просто бери нужное поле, и класс можно переименовать в просто input (тут и в css файле)
```suggestion
export const Input: FC<InputProps> = ({ value, type='addTask' }) => <input value={ value } className={ css.input} />
```